components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponentHolder.java

```diff
@@ -198,4 +200,23 @@ public void setOAuthAdminService(OAuthAdminServiceImpl oauthAdminService) {
 
         this.oauthAdminService = oauthAdminService;
     }
+
+    /**
+     * Set Authentication Data Publisher Proxy instance.
+     *
+     * @param authenticationDataPublisherProxy
+     */
+    public static void setAuthenticationDataPublisherProxy(AuthenticationDataPublisher
+                                                                   authenticationDataPublisherProxy) {
```

Comment:
Add an empty line as the 1st line of the method. Check in other places as well.

Reply:
Modified.